packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -302,7 +318,16 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
     >
       <ComboBoxActionContext.Provider value={dispatch}>
         <ComboBoxStateContext.Provider
-          value={{ id, minWidth, isOpen, focusedIndex }}
+          value={{
+            id,
+            minWidth,
+            isOpen,
+            focusedIndex,
+            enableVirtualization,
+            filteredOptions,
+            getMenuItem,
+            emptyStateContent,
```

Comment:
Now it looks like too much stuff in context. `emptyStateContent` at least could be gotten rid of because we already pass it as `children` of `<ComboBoxMenu>`

Reply:
Removed